### PR TITLE
feat: modify style

### PR DESCRIPTION
**Pull request by QiShaoXuan**
1. Modify font-size, color, and margin with new design

### CODE REVIEW

File: packages/editor/themes/affine.css [hunk 1]
```diff
@@ -1,52 +1,64 @@
...
   --affine-font-family: Avenir Next, Poppins, apple-system, BlinkMacSystemFont,
     Helvetica Neue, Tahoma, PingFang SC, Microsoft Yahei, Arial,
     Hiragino Sans GB, sans-serif, Apple Color Emoji, Segoe UI Emoji,
     Segoe UI Symbol, Noto Color Emoji;
-  --affine-font-mono: Space Mono, Consolas, Menlo, Monaco, Courier, monospace;
+  --affine-font-number-family: Roboto Mono, apple-system, BlinkMacSystemFont,
+    Helvetica Neue, Tahoma, PingFang SC, Microsoft Yahei, Arial,
+    Hiragino Sans GB, sans-serif, Apple Color Emoji, Segoe UI Emoji,
+    Segoe UI Symbol, Noto Color Emoji;
+  --affine-font-code-family: Space Mono, Consolas, Men lo, Monaco, Courier,
```
Comment (lawvs) [inline packages/editor/themes/affine.css:55]:
```suggestion
  --affine-font-code-family: Space Mono, Consolas, Menlo, Monaco, Courier,
```

**Comment by lawvs** [comment]
Test break due to https://github.com/toeverything/blocksuite/issues/902

**Comment by himself65** [comment]
I'm working on this

File: tests/format-quick-bar.spec.ts [hunk 2]
```diff
@@ -35,7 +35,7 @@ test('should format quick bar show when select text', async ({ page }) => {
     throw new Error("formatQuickBar doesn't exist");
   }
   assertAlmostEqual(box.x, 20, 5);
-  assertAlmostEqual(box.y, 260, 5);
```
Comment (himself65) [inline tests/format-quick-bar.spec.ts:38]: Need to conform with @lawvs. What's these number should be?

**Comment by himself65** [comment]
need to correct the format bar test cases /cc @lawvs

File: tests/format-quick-bar.spec.ts (hunk 2 above)
Comment (lawvs) [reply tests/format-quick-bar.spec.ts:38]: This update is ok

**Pull request merged**